Check for deleted meeting and make sure a user never get registered t…

Ash-5 opened the pull request:
…wice for same meeting

Commit:
Check for deleted meeting and make sure a user never get registered twice for same meeting

meyyappanv commented on classes/task/add_meeting_registrant.php:
```diff
@@ -38,62 +38,76 @@ public function execute()
+                                    JOIN mdl_course c ON c.id = ct.instanceid and e.courseid = c.id
+                                    JOIN mdl_role r ON r.id = ra.roleid AND r.shortname = 'student'
+                                    JOIN mdl_zoom mz ON mz.course = c.id
+                            WHERE e.status = 0 AND u.suspended = 0 AND u.deleted = 0
```
@Ash-5 I know that this was already existing query. Just wanted to know of it is better to move the where conditions to the corresponding joins instead of here?

Ash-5 replied:
@meyyappanv I have done the changes can you please check again.

Commit:
Move conditions from where clause to join statement

Commit:
change variable name from findMeeting to meetingWebinarInfo

This change is done because variable will store information related to mentioned meeting or webinar

Commit:
remove role student

This is done so that user with any role withing that perticular course can join meeting even when registration is ON.

Commit:
change variable name from webinar to isWebinar

This is done because it is a boolean value,  which will be wither zero or one

Commit:
change variable from snake case to lower camel case in order to make it look symmetric through out the file
